source/win-preferences/App.vue

```diff
@@ -171,6 +172,12 @@ export default defineComponent({
           controls: 'tab-advanced',
           id: 'tab-advanced-control',
           icon: 'tools'
+        },
+        {
+          label: trans('dialog.preferences.toolbar'),
```

Comment:
Before merging, it would be great if you could list all NEW translation strings that you have added during this PR as an edit to the first comment of the PR, just so that I can easily copy and paste them into Zettlr translate!

Reply:
@nathanlesage Sorry I don't make sure which one is the first comment of the PR..... I have edited the list of translation strings into the change detail of this PR

Reply:
Don't worry, exactly that's where I meant :)